assignments/callbacks.js

```diff
@@ -19,34 +19,66 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
-
 
 function getLength(arr, cb) {
+  return cb(arr.length);
   // getLength passes the length of the array into the callback.
 }
 
 function last(arr, cb) {
+  return cb(arr.length -1)
   // last passes the last item of the array into the callback.
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y + y + y - (2 * y));  
```

Comment:
this is a little too overly engineered here, you should just be adding two numbers and adding it to the callback. This should simply be return cb(x + y).